app/__init__.py

```diff
@@ -0,0 +1,28 @@
+from app.config import Config  # Must be first
+import uuid
+import logging
+
+from flask import Flask
+
+from app.cron.scheduler import Scheduler
+from app.model.model_predict import load_model
+from app.handlers.predict_handler import run_predict
+
+traceId = uuid.uuid4()
+d = {'trace': traceId}
+
+cfg = Config.get_instance().cfg
+logger = logging.getLogger(__name__)
+logger.info(f'Config: {cfg}', extra=d)
+
+app = Flask(__name__)
+app.isof_model = load_model(cfg.model.pathname)
```

Comment:
Check if result is None (failed to load) and exit the application with proper error